aws/database/main.tf

```diff
@@ -66,24 +66,26 @@ resource "aws_security_group" "default" {
 }
 
 resource "aws_db_instance" "default" {
-  allocated_storage       = var.storage_gb
-  storage_type            = var.storage_type
-  engine                  = var.type
-  engine_version          = local.ver
-  instance_class          = var.instance
-  name                    = var.name
-  identifier              = var.identifier
-  username                = local.username
-  password                = random_password.password.result
-  db_subnet_group_name    = aws_db_subnet_group.subnet.id
-  vpc_security_group_ids  = [aws_security_group.default.id]
-  skip_final_snapshot     = "true"
-  publicly_accessible     = var.publicly_accessible
-  multi_az                = var.multi_az
-  ca_cert_identifier      = var.ca_cert_identifier
-  apply_immediately       = var.apply_immediately
-  parameter_group_name    = var.parameter_group_name != "" ? var.parameter_group_name : "default.${var.type}${local.ver}"
-  backup_retention_period = var.backup_retention_period
+  allocated_storage                     = var.storage_gb
+  storage_type                          = var.storage_type
+  engine                                = var.type
+  engine_version                        = local.ver
+  instance_class                        = var.instance
+  name                                  = var.name
+  identifier                            = var.identifier
+  username                              = local.username
+  password                              = random_password.password.result
+  db_subnet_group_name                  = aws_db_subnet_group.subnet.id
+  vpc_security_group_ids                = [aws_security_group.default.id]
+  skip_final_snapshot                   = "true"
+  publicly_accessible                   = var.publicly_accessible
+  multi_az                              = var.multi_az
+  ca_cert_identifier                    = var.ca_cert_identifier
+  apply_immediately                     = var.apply_immediately
+  parameter_group_name                  = var.parameter_group_name != "" ? var.parameter_group_name : "default.${var.type}${local.ver}"
+  backup_retention_period               = var.backup_retention_period
+  performance_insights_enabled          = var.performance_insights_enabled
+  performance_insights_retention_period = var.performance_insights_retention
```

Comment:
Relying on computation of `performance_insights_kms_key_id`?

Reply:
The reason why I haven't add `performance_insights_kms_key_id` is because the module doesn't support RDS encryption (`storage_encrypted`), so I thought wasn't helpful encrypting performance insights and not the database itself.
Do you think we should support this even if not allowing to encrypt the main DB? I can add that portion.
If not, I can follow up with a PR which allows using encryption in both the storage and the performance insights.